.idea/cf-units.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We definitely don't want these files in here. In #130 I ensured ``.idea`` is added to the ``.gitignore``. Could you remove all files from ``.idea`` please.